<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="convert_Btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Convertir</string>
    </property>
   </widget>
   <widget class="QLabel" name="conversion">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pesos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cnt_Base">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="cnt_Objetivo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingrese la tasa de cambio:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tasa_cambio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string> (El valor de venta) (EJ cuantos pesos vale 1 dolar)</string>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QPushButton" name="fijar_taza">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>490</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Fijar taza</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Convirtiendo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="moneda_default">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>310</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Convertir:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Conversion:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="moneda_base">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="moneda_Objetivo">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fijar_moneda">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Fijar moneda</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Monedas disponibles:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>541</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Dolares, Dolar Blue, Pesos, Euros.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
